pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -1436,7 +1262,7 @@ fn can_verify_restriction_with_primary_issuance_agent_we() {
     assert_ok!(verify(issuer_id, random_guy_id), ERC1400_TRANSFER_SUCCESS);
 
     // From primary issuance agent to owner should fail
-    assert_ok!(verify(issuer_id, owner_id), ERC1400_TRANSFER_FAILURE);
+    assert_ok!(verify(issuer_id, owner.did), ERC1400_TRANSFER_FAILURE);
```

Comment:
```suggestion
    assert_ok!(verify(issuer, owner), ERC1400_TRANSFER_FAILURE);
```
(with necessary changes to closure itself)